rqt_top/src/rqt_top/node_info.py

```diff
@@ -53,7 +52,7 @@ def get_node_info(self, node_name, skip_cache=False):
                     return p
                 except:
                     return False
-        except error:
+        except SocketError:
```

Comment:
Oh I see, I missed the missing four spaces :wink: I guess the chance the `socket` within `xmlrpclib` is ever changes is pretty much zero. So this should be good as-is.